socketServer.js

```diff
@@ -88,7 +89,12 @@ SocketServer.prototype.start = function start(server) {
             context.permissions = p;
             resolve(context);
         })))
-        .then((context) => (permissionVerify(context, fingerprint)))
+        .then((context) => {
+            if (this.disablePermissionVerify) {
```

Comment:
This if seems to be wrong.
Even the default disablePermissionVerify: {ws: false} will cause disabling.